docs/tenant_crd.adoc

```diff
@@ -1143,6 +1143,13 @@ Directs the MinIO Operator to use prometheus operator. +
 
 Tenant scrape configuration will be added to prometheus managed by the prometheus-operator.
 
+|*`prometheusOperatorScrapeMetricsPaths`* __string array__ 
+|*Optional* +
+
+
+The name of the Prometheus instance to scrape metrics from.
```

Comment:
Actually prometheus scrapes metrics from underlying system and maintains it. The wordings need to change something like `API end point(s) to scrape metrics from` may be